frontend/src/views/CensusDirectory/components/Project/ProjectButtons/components/EmbeddingButton/connect.ts

```diff
@@ -41,7 +48,7 @@ function pythonCodeSnippet(project: UnionProject, uri: string): string {
       measurement_name = "${measurement}",
       obs_value_filter = "tissue_general == 'central nervous system'",
   )
-  embeddings = get_embedding("${censusVersion}", embedding_uri, adata.obs["soma_joinid"]).tonumpy())
+  embeddings = get_embedding("${censusVersion}", embedding_uri, adata.obs["soma_joinid"]).to_numpy())
```

Comment:
probably will require some research, but it'd be nice to author the file in its native language, so we can leverage linters to catch issues, and then just interpolate the variable values and inject the code as plain text at build time. But right now probably more cost than benefits 😆